e2e-tests/switch-between-services/cypress.config.js

```diff
@@ -9,14 +9,15 @@ dotenv.config();
  * Cypress configuration
  */
 const cypressConfig = defineConfig({
+  video: true,
```

Comment:
`video`? Same above

Reply:
This is for local tests only, when they fail I can see the recording. This is overwritten during CLI execution on CI.